tests/js/getsentry-test/fixtures/am1Plans.ts

```diff
@@ -57,7 +57,6 @@ const AM1_TEAM_FEATURES = [
   ...AM1_FREE_FEATURES,
   'codecov-integration',
   'crash-rate-alerts',
-  'dashboards-basic',
```

Comment:
Should these features removed from the fixtures? Flagpole will add them to the features.

Reply:
@vbro yeah we want to remove these because ultimately the goal is to remove them from the `getsentry` plan definitions but we need to bridge the period where we haven't released these flags to all plans yet and the flags are controlled by flagpole. Removing them from the fixtures essentially replicates that state because even if the fixture doesn't have the flag, `sentry` still knows which plan is minimally required to get that feature

if I understand correctly, Flagpole is just an evaluation if the current user/org satisfies the conditions for the flag, but it won't "inject" the feature flag into the corresponding plan, which is what makes this PR necessary to keep the correct upsells